pkg/jobs/registry.go

```diff
@@ -1282,6 +1286,33 @@ func (r *Registry) adoptionDisabled(ctx context.Context) bool {
 	return false
 }
 
+// A single job should not toggle its idleness more than twice per-minute as it
+// is logged and may write to persisted job state in the future
```

Comment:
This was from the discussion in https://github.com/cockroachdb/cockroach/issues/74747, I don't mind removing the persistence/freq specifics since the main point is just the infrequent requirement

Reply:
APIs have contracts that are just in comments, and maybe not _enforced_ in code but could cause misbehavior if violated, all the time like "caller must hold lock xyz" or "is not threadsafe". I think "should not be called more than x times per minute" fits that bill too (and is how we justify logging below)

Reply:
nit: comment should end with a period.